HomeTask_10_dir/HomeTask_10.py

```diff
@@ -0,0 +1,10 @@
+import re
```

Comment:
Please do not forget about file docstring with task description. Also leave comments in your code for key moments